bitsail-common/src/main/java/com/bytedance/bitsail/common/type/filemapping/MongoTypeInfoConverter.java

```diff
@@ -111,7 +121,12 @@ public TypeInfo<?> getTypeInfo(String type) {
   }
 
   public TypeInfo<?> getBasicTypeInfoFromMongoDBType(String mongoType) {
-    return reader.getToTypeInformation().get(mongoType);
```

Comment:
suggest to add some ut test

Reply:
> @beyond-up Can we add more check after the code both in the `NativeFlinkTypeInfoUtil` and `ColumnFlinkTypeInfoUtil`
> 
> ```
>   public static RowTypeInfo getRowTypeInformation(List<ColumnInfo> columnInfos,
>                                                   TypeInfoConverter typeInfoConverter) {
> 
>     String[] fieldNames = new String[columnInfos.size()];
>     TypeInformation<?>[] fieldTypes = new TypeInformation[columnInfos.size()];
> 
>     for (int index = 0; index < columnInfos.size(); index++) {
>       String type = StringUtils.lowerCase(columnInfos.get(index).getType());
>       String name = columnInfos.get(index).getName();
> 
>       TypeInfo<?> typeInfo = typeInfoConverter.fromTypeString(type);
> 
>       //in here
>       fieldNames[index] = name;
>       fieldTypes[index] = toNativeFlinkTypeInformation(typeInfo);
>     }
> 
>     return new RowTypeInfo(fieldTypes, fieldNames);
>   }
> ```
> 
> Not every new type converter has enough validation rules, so add validation check after the `typeInfoConverter` invoke maybe more make sense? What do you think.

OK, no problem, then I will remove the exception information in the existing converter  like this

Reply:
> suggest to add some ut test

ok, i will add @lichang-bd